bot.py

```diff
@@ -436,6 +437,27 @@ def check(answer):
                             pass
         await asyncio.sleep(1)
 
+@asyncio.coroutine
+async def update_game_channels():
+    guild = get_guild()
+    while True:
+        for channel in guild.voice_channels:
+            if re.match(r"[WG]\d", channel.name):
```

Comment:
A comment to explain what this regex is matching against would be helpful, since regex are sometime dark magic.